tribe-ext-selectively-show-html/src/functions/general.php

```diff
@@ -0,0 +1,59 @@
+<?php
```

Comment:
The one with tribe_call_func() was a relic from the snippets framework. This used to be built atop that. Thanks for pointing this out, that file is now deleted.